### ISSUE TITLE
Why is st-ReID + RE lower than in the paper?

### ISSUE THREAD

**Issue by Adorablepet** [Wanggcong/Spatial-Temporal-Re-identification#8]
environment: pytorch 1.0/python 3.6/one-gpu
I train st-Reid+RE model according to the parameters you code provide.(original parameter)
st-Reid+RE
map:87%, top1:96.7%
st-Reid+RE + re-rank
map: 94.5%, top1:96.8%

**Comment by Wanggcong** [comment]
Our baseline (appearance feature) is ~91%. It seems that your baseline is not so good?

**Comment by Adorablepet** [comment]
@Wanggcong My baseline is provided in your code. I have not modified it.I clone your code to train with original parameter.I'm confused.

**Comment by Wanggcong** [comment]
The reason may be attributed to different versions of pytorch. We will provide the trained model using this code.

**Comment by Adorablepet** [comment]
@Wanggcong Another problem is that when I use the mixed-precision to train the model, there will be a "gradient overflow". Is there a way to solve this?according to the way with fp16 layumi provides.
val_loss: acc: 1.0
